lib/wasm-interface/Cargo.toml

```diff
@@ -1,12 +1,14 @@
 [package]
 name = "wasmer-wasm-interface"
 version = "4.0.0-alpha.1"
```

Comment:
Why keeping this one?

Reply:
Good catch! I'm guessing I skipped that one because I thought it was one of the packages that gets versioned separately.